science/xraylib/Portfile

```diff
@@ -53,11 +53,11 @@ configure.ccache    no
 
 perl5.require_variant yes
 perl5.conflict_variants yes
-perl5.branches 5.28 5.30
-perl5.default_branch 5.28
+perl5.branches 5.32 5.34
+perl5.default_branch 5.32
```

Comment:
unless a port doesn't work with it, the default branch should remain the same as the one that is set in the `perl` PG, which is currently 5.28.

Reply:
Oh. I read 5.30 and earlier were unsupported, but I don't know anything about Perl. I'll add all of 5.28, 30, 32, and 34 with 5.28 the default.

Reply:
Done.